src/TaskRunner/Commands/ToolCommands.php

```diff
@@ -844,14 +844,14 @@ public function optsReview()
                     $command = str_replace('\\', '', $command);
                     foreach ($forbiddenCommands as $forbiddenCommand) {
                         if ($key == 'default') {
-                            $parsedCommand = explode(" ", $command);
+                            $parsedCommand = preg_split("/[\s;&|]/", $command, 0, PREG_SPLIT_NO_EMPTY);;
```

Comment:
Hi @cherouvim ,
There's a double ";", please remove one of them.
Best regards

Reply:
Oops, sorry, fixed.